client/components/reviewWidget/ReviewCard.jsx

```diff
@@ -1,36 +1,60 @@
 import React from 'react';
 import StarAverage from '../shared/StarAverage.jsx';
 
-const ReviewCard = () => {
+const ReviewCard = ({ review, index }) => {
+  // Conditionally render a response if present
+  let response = null;
+  if (review.response) {
+    response = (
+      <div className="review-response">
+        <h6>Response:</h6>
+        <p>{review.response}</p>
```

Comment:
I think this be {review.body}. Same for line 7. I don't see these elements on the page, probably for this reason.